htdocs/hrm/position_list.php

```diff
@@ -218,6 +219,10 @@
 // --------------------------------------------------------------------
 $sql = 'SELECT ';
 $sql .= $object->getFieldList('t');
+$sql .= ',';
+$sql .= $userstatic->getFieldList('u');
+$sql .= ',u.email, u.statut';
```

Comment:
to remove because it's in getFieldList

Reply:
the fields in class user doesn't have an email and statut